modules/servers/pvewhmcs/pvewhmcs.php

```
@@ -1257,6 +1260,7 @@ function pvewhmcs_noVNC($params) {
 
 // VNC: Console access to VM/CT via SPICE
 function pvewhmcs_SPICE($params) {
+	global $CONFIG:
```

PHPStan (2.1.46)

[error] Syntax error, unexpected ':', expecting ';' on line 1263


(phpstan.parse)